coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 46.8,
+  "coverage_score": 46.1,
   "exclude_path": "",
   "crate_features": ""
 }
```

Comment:
ok no reason to hold up this PR then.

Reply:
I will push this after https://github.com/rust-vmm/vhost-device/pull/62 gets merged